app/src/main/res/values/strings.xml

```diff
@@ -438,6 +438,10 @@
     <string name="report_send_success">Skill reported successfully</string>
     <string name="reported_already">Skill already reported</string>
 
+    <!--Default placeholder values of skills description-->
+    <string name="default_author_name" translatable="false">&lt;author_name&gt;</string>
+    <string name="default_example" translatable="false">&lt;The question that should be shown in public skill displays&gt;</string>
```

Comment:
Remove from strings and make String constants

Reply:
I thought of that, but the same problem exists in the skill details fragment. So I thought these strings might be useful in the future. Please enlighten, if asked changes should be done or not.

Reply:
They should be done